standard-covoiturage_openapi.yaml

```diff
@@ -176,6 +176,171 @@ paths:
         500:
           $ref: '#/components/responses/InternalServerError'
 
+  /messages:
+    post:
+      tags:
+      - Interact
+      summary: Send a message to the owner of a retrieved journey.
+      description: Route used to allow a user to connect back to the owner of a retrieved journey through a texte message.
+      operationId: postConnections
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/ConnectionMessage'
+              type: object
+              required:
+                - message
+                - recipientCarpoolerType
+              properties:
+                recipientCarpoolerType:
+                  type: string
+                  enum:
+                    - driver
+                    - passenger
+                  default: driver
+                  description: Defines if the recipient of this message is either the driver or the passenger.
+      responses:
+        201:
+          description: Successful operation.
+        404:
+          description: The targeted journey or user no more exists.
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        429:
+          $ref: '#/components/responses/TooManyRequests'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+  /bookings:
+    post:
```

Comment:
**Review on the POST /booking endpoint :** 
- The `message` parameter should not be compulsory.
- Why having added a `type` parameter to the booking ? It seems irrelevant to me.
- Why having added a `userIdToken` for MaaS Connect since the core objective of the booking by API is to book a journey _without_ having to connect to another service ?

Reply:
> The message parameter should not be compulsory.
Ok

> type parameter to the booking ? It seems irrelevant to me.
It came as a byproduct of puting a whole `JourneySchedule` into the booking. I'll think about how to reorganize things.

> Why having added a userIdToken for MaaS Connect since the core objective of the booking by API is to book a journey without having to connect to another service ?

Ok, with this comment + the comment on the price, I think I get the idea. In the booking by API, the carsharing operator doesn't have to know the details about the passenger (so, no specific authentification).